src/SmartComponents/DriftPage/DriftTable/DriftTable.js

```diff
@@ -38,6 +38,19 @@ export class DriftTable extends Component {
         this.removeSystem = this.removeSystem.bind(this);
     }
 
+    doubleScroll() {
+        let wrapper1 = document.getElementById('top-scroll');
+        let wrapper2 = document.getElementById('table-wrapper');
```

Comment:
React has a ref support on components that allow to access variable-assigned components. Read more here https://reactjs.org/docs/refs-and-the-dom.html

This approach does not allow to have more instances. It expects unique ids. If a raw JS approach is needed then a relative element find would be more proper. Why not leverage what the React framework offers :slightly_smiling_face:.